dcos_installer/upgrade.py

```diff
@@ -58,35 +49,44 @@
     fi
 fi
 
+SKIP_CHECKS=false
+
+if [[ $# -ne 0 ]]; then
+  if [[ "$1" = "skip-checks" ]]; then
+     echo "Skipping checks"
+     SKIP_CHECKS=true
+  fi
+fi
+
 if [[ $EUID -ne 0 ]]; then
     echo "This script must be run as root" 1>&2
     exit 1
 fi
 
 # check for version of dc/os upgrading from
 found_version=`grep "version" /opt/mesosphere/etc/dcos-version.json | cut -d '"' -f 4`
-if [ "$found_version" != "{{ installed_cluster_version }}" ]; then
+if [[ "$found_version" != "{{ installed_cluster_version }}" ]]; then
     echo "ERROR: Expecting to upgrade DC/OS from {{ installed_cluster_version }} to {{ installer_version }}." \
          "Version found on node: $found_version"
     exit 1
 fi
 
-# Check if the node has node/cluster checks and run them
-if [ -f /opt/mesosphere/etc/dcos-diagnostics-runner-config.json ]; then
-    # command exists
-    output=$(dcos-diagnostics check node-poststart)
-    if [ $? -ne 0 ]; then
-        echo "Cannot proceed with upgrade, node checks failed"
-        echo $output
-        exit 1
-    fi
-
-    clusteroutput=$(dcos-diagnostics check cluster)
-    if [ $? -ne 0 ]; then
-        echo "Cannot proceed with upgrade, cluster checks failed"
-        echo $clusteroutput
-        exit 1
-    fi
+if [[ "$SKIP_CHECKS" = false ]]; then
+   # Check if the node has node/cluster checks and run them
+   if [ -f /opt/mesosphere/etc/dcos-diagnostics-runner-config.json ]; then
+      # command exists
+      if ! output="$(dcos-diagnostics check node-poststart 2>&1)"; then
+          echo "Cannot proceed with upgrade, node checks failed"
```

Comment:
do you want these echoes to go to stdout or etderr? Usually erro messages should go to stderr, unless you have a good reason not to.
```
echo >&2 "Cannot proceed with upgrade, node checks failed"
echo >&2 "$output"
```